### PR TITLE
gst-plugins-bad: Allow building the gstgtk plugin

### PR DESCRIPTION
**Pull request by aleb**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Does your submission pass `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)? 

```
gst-plugins-bad:
 * A `test do` test block should be added
 * Description is too long. "name: desc" should be less than 80 characters.
Length is calculated as gst-plugins-bad + desc. (currently 83)
```
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

**Comment by apjanke** [comment]
Would you mind adding a test and fixing the `desc` while you're in here?

File: Formula/gst-plugins-bad.rb [hunk 1]
```diff
@@ -55,6 +56,8 @@ def install
       system "./autogen.sh"
     end
 
+    args << "--with-gtk=3.0" if build.with? "gtk+3"
```
Comment (DomT4) [inline Formula/gst-plugins-bad.rb]: Nit, but please shuffle this up so it's underneath the other `args <<` line rather than down here by itself.

**Comment by aleb** [comment]
Even with "GStreamer plugins (less supported)" it complains (83 chars). That rule is not reasonable from my point of view, at least as long as the formula name is included.

File: Formula/gst-plugins-bad.rb (hunk 1 above)
Comment (aleb) [inline Formula/gst-plugins-bad.rb]: Done

**Comment by aleb** [comment]
Not sure what test would be appropriate.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/3311ad25cfd4d95d408b3e4d109dd51701dcc7f1. Thank you for your contribution to Homebrew @aleb; we appreciate it! :smiley_cat:

**Pull request closed without merging**